.travis.yml

```diff
@@ -14,6 +14,7 @@ matrix:
     - os: linux
       jdk: openjdk8
     - os: osx
+      osx_image: xcode9.3
```

Comment:
@valib It turns out 9.3 wasn't the best choice since it will soon be removed, so the previous 8.x default is probably a better choice.

Reply:
@Nadahar I rely on your macOS knowledge so what you reccomend in this case I will follow :-)

Reply:
@valib Yes, @Sami32 made me aware that 9.3 wasn't the smartest choice after I did it, which is why I mentioned it. It doesn't relate to macOS per se, it's just that Travis CI has tied the Java version to the xcode version for some reason. 9.4, which is the new default, has Java 10, while the earlier images has Java 8.

Reply:
@Nadahar we should decide to use either xcode9.2 or xcode8.3 because we will not pass the Travis test until we fully implement Java 10 e.g. new LookAndFeel for Windows and other.

Reply:
@valib Yes, that was my point to. 8.3 might be the best choice, it all depends on which version will stay available the longest at Travis CI. All that matters to us is the Java version, as we don't use xcode itself.